These test requirements are either optional or not needed to run tests.
We do not have them packaged.

--- nox-2024.3.2/requirements-test.txt.orig
+++ nox-2024.3.2/requirements-test.txt
@@ -1,10 +1,5 @@
 coverage[toml]>=5.3
 flask
-myst-parser
 pytest>=6.0
 pytest-cov
 sphinx>=3.0
-sphinx-autobuild
-sphinx-tabs
-uv; python_version>='3.8'
-witchhazel
